src/styles/components/_code-pattern.scss

```diff
@@ -26,12 +26,14 @@
   pre {
     border: 0;
     background: 0;
+    white-space: pre-wrap;
```

Comment:
I don't think we want this to pre-wrap. I think we want it to scroll horizontally.

Reply:
How to achieve it without setting the explicit width of the tabs (we'd like it to be 50% of the component)?

Reply:
This is a weird quirk of grid. You have to define a minimum width for the columns.
https://css-tricks.com/preventing-a-grid-blowout/

```
grid-template-columns: minmax(0, 1fr) minmax(0, 1fr)
```

Then you'd remove the `white-space` property from the pre element and it should start displaying a scrollbar.

![image](https://user-images.githubusercontent.com/1066253/129622285-694f8c86-2cac-4d98-a073-48c74f17201d.png)

However, if there are multiple lines of code then we'll end up with both a horizontal and vertical scrollbars, which isn't great...

I noticed that CodePen wraps the text, but you can also drag to resize the container to read more.
![image](https://user-images.githubusercontent.com/1066253/129623183-2f7c4f63-8d43-4cd1-9f4e-83ba8b663c93.png)

I tried putting `position: relative; resize: both` on the `pre` element but it pushed around other elements on the page when I tried to resize it. My thinking is maybe we could leave pre-wrap if we can figure out a way to let the user resize the code? We may try asking Una to help with that CSS

Reply:
We need to make demo resizeable so why not both sides.

Reply:
Issue: https://github.com/GoogleChrome/web.dev/issues/5988